include/mesh/mesh_function.h

```diff
@@ -123,6 +123,15 @@ class MeshFunction : public FunctionBase<Number>,
                      const Real time=0.) libmesh_override;
 
   /**
+   * @returns a map of values of variable 0 at point
+   * \p p and for \p time, which defaults to zero.
+   * map is from element on Number and accounts for double defined
+   * values on faces when discontinuous variables are used
+   */
+  std::map<const Elem *, Number> discontinuousValue (const Point & p,
```

Comment:
I think the `which defaults to zero.` refers to `time=0`. The return value default (`out_of_mesh_value`) can be set.